package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "resin-typescript-skeleton",
+  "version": "0.0.1",
+  "description": "Skeleton template for a Resin TypeScript project",
+  "homepage": "https://github.com/resin-io/resin-typescript-skeleton#readme",
+  "preferGlobal": true,
+  "types": "build/**/*.d.ts",
+  "keywords": [
+    "resin",
+    "typescript"
+  ],
+  "author": "Heds Simons <hedley@resin.io>",
+  "license": "Apache-2.0",
+  "main": "./build/implement-me.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/resin-io/resin-typescript-skeleton.git"
+  },
+  "bugs": {
+    "url": "https://github.com/resin-io/resin-typescript-skeleton/issues"
+  },
+  "scripts": {
+    "build": "gulp build"
+  },
+  "devDependencies": {
+    "@types/node": "^6.0.48",
```

Comment:
I've taken this to be a CLI, hence in `devDependencies`. I think this should therefore be in the `README.md` as a 'How to alter this project depending on use case`, maybe.